### ISSUE TITLE
copy legend to clipboard not work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#194]
```
Nothing happens when you copy a map legend to clipboard (right-click option). 
Copy map as image does work.
This doesn't work in 1.6 or 1.7.
```

Original issue reported on code.google.com by `jkosc...@asu.edu` on 9 Jul 2015 at 10:17

**Comment by lixun910** [comment]
see previous comment. 

Please mark to won't fix.

**Comment by jkoschinsky** [comment]
GeoDa 1.7.63 Mac OSX 10.7.5

Not working yet: When I right-click on legend Copy Legend to Clipboard and then paste it into Preview, Word, etc.: Nothing happens.

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.85 (OSX Yosemite, 10.10.5 and Win 7 Professional (64-bit)).
